cassovary-core/src/test/scala/com/twitter/cassovary/util/io/AdjacencyListGraphReaderSpec.scala

```diff
@@ -42,6 +42,11 @@ class AdjacencyListGraphReaderSpec extends WordSpec with Matchers with GraphBeha
     val graph = reader.toSharedArrayBasedDirectedGraph()
   }
 
+  trait ArrayBasedDynamicGraphWithoutRenumberer {
+    val graph = AdjacencyListGraphReader.forIntIds(directory, "toy_6nodes_adj",
+      Executors.newFixedThreadPool(2)).toArrayBasedDynamicDirectedGraph()
```

Comment:
not sure how this compiles as the third argument "Executors..." was deprecated in master. Please try running test on your latest commit.